Form-Controls/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  width: 100%;
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: 0;
+  background: rgb(245, 206, 218);
+  padding: 0;
+}
+
+
+h1{
+  
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+input{
+  width: 400px;
```

Comment:
a fixed width of 400px might be problematic on smaller screens. Consider using a percentage-based width or max-width to ensure responsiveness.